staff/kim-garcia/cv/index.html

```diff
@@ -1,94 +1,128 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>CV F.L.C.</title>
-    <link rel="stylesheet" href="./styles.css">
-</head>
-<body>
-    <h1>CURRICULUM VITAE</h1>
-    <h2>FERNINANDO LEOPOLDO CASAMAYOR</h2>
-
-<table>
-<tr>
-    <td>
-        <ul>
-            <li>Fer para los amigos</li>
-            <li>Domicilio: Ciudad de Kiribati</li>
-            <li>Fecha de nacimienti: 07/10/1991</li>
-            <li>DNI: 1234567890Z</li>
-            <li>Telefono: 7894568</li>
-            <li>Email: chamuyador@mail.ki</li>
-        </ul>     
-    </td>
-    <td>
-        <img src="fer.jpg" alt="foto">
-    </td>
-</tr>
-
-
-
-
-</table>
-  
-
-    <table style="width:100%">
-        <tr>
-            <td>Fer para los amigos</td>
-        </tr>
-        <tr>
-          <td>Domicilio: Ciudad de Kiribati</td>
-        </tr>
-        <tr>
-          <td>Fecha de nacimienti: 07/10/1991</td>
-        </tr>
-        <tr>
-            <td>DNI: 1234567890Z</td>
-        </tr>
-        <tr>
-            <td>Telefono: 7894568</td>
-        </tr>
-        <tr>
-            <td>Email: chamuyador@mail.ki</td>
-        </tr>
-      </table>
-      <p> 
     
-    </p>
-      <table>
-          <tr>
-              <th>ESTUDIOS</th>
-          </tr>
-          <tr>
-            <td>Primarios en la escuela los Geronimos</td>
-        </tr>
-        <tr>
-            <td>Universidad de la vida</td>
-        </tr>
-      </table>
-
-
-      <h4>IDIOMAS</h4>
-
-      <ul>
-        <li>Catalan</li>
-        <li>Suahili</li>
-        <li>Kiribateño autoctono</li>
-      </ul>  
-<p>
-
-</p>
-      <h4>EXPERIENCIA LABORAL</h4>
-
-      <ul>
-        <li>Guia turistica en empresa de caza</li>
-        <li>Marinero de agua dulce</li>
-        <li>Comercial de enciclopedias</li>
-      </ul>  
-
+</head>
 
+<body>
+    <header>
+        <hgroup>
+                <h1>CURRICULUM VITAE</h1>
+                <h2><cite>“Hay una fuerza motriz más poderosa que el vapor, la electricidad y la energía atómica: la voluntad”</cite></h2>
+        </hgroup>
+        <hr>
+    </header>
+    <main>
+        <section>
+                <h2>FERNINANDO LEOPOLDO CASAMAYOR</h2>
+                <article>
+                        <figure>
+                                <img src="fer.jpg" alt="foto de leo">
+                        </figure>
+                        <table>
```

Comment:
Perhaps you could use <ul> and <li> instead of a table.

Reply:
agree: for lists use lists, not tables

Reply:
done